design/Bootstrapping_strategy.md

```diff
@@ -134,14 +134,14 @@ Our business requirement is to respond to users with a "soft finality" in no mor
 Note: That might be a bit ambitious given there are network latencies involved.
 
 The other requirement is that all L2 nodes must act as read-only replicas of the SA, able to serve data requests on behalf of the SA.
-Given that they are owend by byzantine hosts, the L2 node must only execute the transaction batches that are originating from the SA. 
+Given that they are owned by byzantine hosts, the L2 node must only execute the transaction batches that are originating from the SA. 
 If they were able to execute anything else, they could malicously produce wrong results.
 
 Users submit transactions to any L2 node, who gossip them among themselves, in a group that also includes the SA. (same as today)
 
 Every second the enclave of the SA produces a signed "Light Batch" (LB), which is a list of tx hashes, with a header pointing to the previous batch.
 This LB is encrypted, and gossiped to all the other nodes.
-Note: The enclave can only produce it once it calculated a chain of N hashes ( roughtly a second worth). This is necessary to prevent frontrunning by the SA. Basically the timer is not the clock of the host, but a mechanism based on a light proof of work.
+Note: The enclave can only produce it once it calculated a chain of N hashes (roughly a second worth). This is necessary to prevent frontrunning by the SA. Basically the timer is not the clock of the host, but a mechanism based on a light proof of work.
```

Comment:
Were there any other options considered for the timing? I guess this was the best or only way?

Reply:
I was just correcting a typo, don't want to drag this doc into this PR review please :)

Reply:
Yeah sorry, I realised I meant to ask it in our meeting just now, should have just put it in discord.